front/public/cgi-bin/printenv.sh

```diff
@@ -0,0 +1,9 @@
+404: Script Not Found
+===================
+
+The script you're looking for was not found OR YOU WERE TRYING TO HACK MY SITE.
+
+NICE TRY. I HAVE LOGGED YOUR IP ADDRESS AND YOU WILL BE HACKED!!1
+
+
+Please report vulnerabilities at [the issue tracker](https://github.com/dikaiosune/rust-dashboard/issues).
```

Comment:
So I'm not sure how comfortable I am sticking files with the `.sh` extension in the public asset folder, even if it is ostensibly non-executable and for a good reason :). Also, the ember app does have a prominent GitHub ribbon which I assume a well-intentioned security researcher could use to find the repository or my GitHub profile for reporting issues.

While I'm nitpicking, at some point there's a chance that this code will live on a server funded by Mozilla, and so I think it'd be good to maintain a professional tone throughout.